### PR TITLE
Emojis!

### PR DESCRIPTION
**Pull request by JCB-K**
remove unused code, move JS to separate file, render emoji based on face expression

ping @jesperp @thibautRe

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,59 @@
...
+  // excuse the verbose code 😁
+  if ($.inArray(data.angerLikelihood, likely) != -1) {
+    emojiVersionOfImage ='😡';
+  } else if ($.inArray(data.joyLikelihood, likely) != -1) {
+    emojiVersionOfImage ='😀';
+  } else if ($.inArray(data.sorrowLikelihood, likely) != -1) {
+    emojiVersionOfImage ='😟';
+  } else if ($.inArray(data.surpriseLikelihood, likely) != -1) {
+    emojiVersionOfImage ='😮';
+  }
```
Comment (jesperp) [inline src/index.js:58]: @JCB-K maybe add a default ":expressionless:"

Comment (JCB-K) [inline src/index.js:58]: ah yeah you're right, forgot that one

**Comment by jesperp** [comment]
Good job @JCB-K !

`README` is referring to index.html for `API_KEY`, can u update that?

**Comment by jesperp** [comment]
gonna keep emoji as suggestion, closing this one down

**Pull request closed without merging**